lang/fpc/Portfile

```diff
@@ -293,6 +312,16 @@ if {${subport} eq "${name}"} {
                "bin/fpcmkcfg -d basepath=${fpcbasepath}/lib/${name}/${version} -o etc/fpc.cfg"
         ln -s ${fpcbasepath}/etc/fpc.cfg ${destroot}${prefix}/etc
         system "patch ${destroot}${fpcbasepath}/etc/fpc.cfg ${filespath}/fpc.cfg.patch"
+        # remove the -WM option for older systems or update it to ${macosx_deployment_target}
+        if {${os.major} < 11} {
+            # os.major = 11 is MacOSX = 10.7 (Lion)
+            reinplace "s|-WM10.9||g" ${destroot}${fpcbasepath}/etc/fpc.cfg
```

Comment:
Is there some reason not to use deployment target on < 10.7?

Reply:
I forgot the exact details, but as much as I can remember, at least the switch was not required for older systems. It might even cause non-helpful warnings or even errors because of the limitation to a particular system version. It is just a kind of caution from my side, not to introduce a patch for systems, where they are not required. But I have no older systems to test. So, if you find out that it is not causing problems, we might skip this.

Reply:
Ok, got it. I think it is easier if I add PPC support all at once in a PR following this one, and for that I can test 10.5 and 10.6. So whatever settings you consider reasonable at the moment will do. Then I can test the build with default settings, and if something fails, we can find a solution.